index.html

```diff
@@ -0,0 +1,364 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <!-- favicon dedicated to different browsers - START -->
+    <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png" />
+    <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png" />
+    <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png" />
+    <link rel="manifest" href="/site.webmanifest" />
+    <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#5bbad5" />
+    <meta name="msapplication-TileColor" content="#da532c" />
+    <meta name="theme-color" content="#ffffff" />
+    <!-- favicon dedicated to different browsers - STOP -->
+
+    <link rel="stylesheet" href="./styles/reset.css" />
+    <link rel="stylesheet" href="./styles/index.css" />
+
+    <title>Nova - Beautiful Free Template</title>
+  </head>
+
+  <body>
+    <header class="main-header">
+      <div class="navbar">
+        <div class="logo">
+          <a class="logo__link logo__link--hover" href="./index.html">
+            <img
+              class="logo__image"
+              src="./images/logo.svg"
+              alt="Logo of Nova"
+            />
+          </a>
+        </div>
+
+        <nav class="menu">
+          <ul class="menu__item-list">
+            <li class="menu__item menu__item--hover">

```

Comment:
Skoro one wszystkie maja modyfikator o nazwie `--hover` to jaki jest sens jego stosowania jak wszystkie mają tą klasę? Nie wystarczy element tj. `menu__item`?